paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -90,24 +79,14 @@ internal class DefaultFlowController @Inject internal constructor(
     private val context: Context,
     private val eventReporter: EventReporter,
     private val viewModel: FlowControllerViewModel,
-    paymentLauncherFactory: StripePaymentLauncherAssistedFactory,
-    /**
-     * [PaymentConfiguration] is [Lazy] because the client might set publishableKey and
-     * stripeAccountId after creating a [DefaultFlowController].
-     */
-    lazyPaymentConfiguration: Provider<PaymentConfiguration>,
+    private val confirmationHandler: ConfirmationHandler,
```

Comment:
`DefaultFlowController` is a special case in which we can inject the `ConfirmationHandler` directly since we create the `FlowControllerViewModel` first.